pkgs/by-name/to/todds/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  boost,
+  fmt,
+  hyperscan,
+  ispc,
+  opencv,
+  tbb_2021_11,
+}:
+stdenv.mkDerivation rec {
+  pname = "todds";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "todds-encoder";
+    repo = "todds";
+    rev = version;
+    hash = "sha256-nyYFYym9ZZskkaTPV30+QavdqpvVopnIXXZC6zkeu7c=";
+    fetchSubmodules = true;
```

Comment:
Can we use the vcpkg from nixpkgs instead?

Reply:
turns out it wasn't needed at all :p didn't think to check if the submodule was actually necessary for the  build